styles/main.css

```diff
@@ -236,34 +236,35 @@ img {
     header nav ul a{
         border:0;
     }
-    header nav ul a :hover {
+    header nav ul a:hover {
+        border: 0;

```

Comment:
You need to remove both the default transparent border (which you did) AND the hover border (missing in your code)